<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html>
<head>




<title> H201SC20021239_X201SC20021239-Z01-F001_Coriandrum Plant and Animal Resequencing Project Analysis Report </title>



<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />



<link rel="stylesheet" href="src/css/jquery-ui.css" />

<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />

<link rel="stylesheet" href="src/css/jquery.tocify.css" />

<link rel="stylesheet" href="src/css/style.css"/>

<link rel="stylesheet" href="src/css/base.css"/>

<link rel="stylesheet" href="src/css/gallery.css"/>

<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>

<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>

<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>

<script src="src/js/scrollTop.js" type="text/javascript"></script>

<script type="text/javascript">

        $(function () {

                var options = {

                        selectors: "h2,h3,h4,h5",

                        theme: "bootstrap",

                        scrollTo: "60"

                };

                var toc = $("#toc").tocify(options).data("toc-tocify");

        });

</script>





<script type="text/javascript">

function trans(img_div_id) {

img_div = $("#" + img_div_id)

img = $("#" + img_div_id + " img").first();

if (img_div.hasClass("fullscreen-div")) {

img_div.removeClass("fullscreen-div");

img.removeClass("fullscreen-img");

}

else {

img_div.addClass("fullscreen-div");

img.addClass("fullscreen-img");

}

}



function filter_display() {

return $(this).css("display") != "none";

}



function show_default() {

var list_group_id = "#" + $(this).attr("id");
if ($(list_group_id + " a.active").css("display") == "none") {

var list_tab = $(list_group_id + " a.active");

var pan_id = list_tab.attr("href");

$(pan_id).removeClass("active show");

list_tab.removeClass("active");

}

if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

$(list_group_id + " a").filter(filter_display).first().tab("show");

}

}

$.fn.show_default = show_default



function next(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

while (present_tab.next().length == 1) {

if (present_tab.next().css("display") != "none") {

present_tab.next().tab("show");


break;

}

else {

present_tab = present_tab.next();

}

}



}

function prev(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

while (present_tab.prev().length == 1) {

if (present_tab.prev().css("display") != "none") {

present_tab.prev().tab("show");

break;

}

else {

present_tab = present_tab.prev();

}

}





}



$(document).ready(function () {

$(".form-control").on("keyup", function () {

var value = $(this).val().toLowerCase();

var input_box_id = "#" + $(this).attr("id");

var list_group_id = input_box_id.replace("_InputList", "_list-tab");

$(list_group_id + " a").filter(function () {

$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

});

$(list_group_id).show_default();

});
$(".list-group").each(show_default);

});


</script>

</head>













<body>
<div id="toc" class="tocify">

</div>





        <div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

                <table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"

                        style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">

                        <tr height="86%">

                                <td width='1%'></td>

                                <td><img src="src/images/logo1.png"></td>

                                <td>

                                        <div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div>

                                </td>

                                <td width="20%"></td>

                                <td width="15%"></td>

                                <td width="25%"

                                        style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">

                                        Tel: 916-252-0068-383

                                        <br><a href="mailto:op-rna@novogene.com" target="black"

                                                style="text-decoration:none;color:black;font-size:14px;font-weight:50;">Email: inquiry_us@novogene.com

                                                <a></td>

                                <td width="1%"></td>

                        </tr>

                </table>

        </div>


</div>



<!--------------------------标题---------------------------------->

<div id="main">




</br><h1> Plant and Animal Resequencing Project Analysis Report </h1></br>



<table class= 'gy' style="text-align:left">

<thead><tr><th>Contract Information</th><th>Contract Content</th></tr></thead>

<tbody>



<tr>



<td>Contract ID</td>



<td>H201SC20021239</td>



</tr>



<tr>



<td>Batch ID</td>



<td>X201SC20021239-Z01-F001</td>



</tr>



<tr>



<td>Species and Version</td>



<td>Coriandrum</td>



</tr>



<tr>



<td>Report Time</td>



<td>2020-03-19</td>



</tr>



</tbody></table>

<br /><br />


<!---------------------------Experimental Process--------------------------------------------------->
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Library_Preparation_and_Sequencing">Novogene Co., Ltd</a>
<hr />
</p><br />
<h2>1 Workflow of Resequencing Project</h2>
<p class="paragraph">Plant and animal whole genome resequencing via Illumina platforms, based on mechanism of SBS (sequencing by synthesis), has become the most rapid and effective method to identify the genetic variations in individuals of the same species or between related species. The variation information such as Single Nucleotide Polymorphism (SNP), Insertion and Deletion (InDel), Copy Number Variation (CNV), and structural variation (SV) obtained through resequencing is used in population genetics research and genome-wide association studies (GWAS) to investigate the causes of diseases, to select plants and animals for agricultural breeding programs, and to identify common genetic variations among populations. Plant and animal whole genome resequencing projects are carried out as follows: </p>
<p class="center">
<img class="small2" src="src/images/experimentWorkflow.png" />
</p>
<p class="name_fig">Figure 1.1 Workflow of Resequencing Project</p>


<a name="DNA_Quantification_and_Qualification"></a>
<h3>1.1 DNA Quantification and Qualification</h3>
<p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;(1)&nbsp;DNA degradation and contamination were monitored on agarose gel electrophoresis analysis;</p>
<p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;(2)&nbsp;Qubit<sup>®</sup> DNA concentration was measured using Qubit® DNA Assay Kit in Qubit® 2.0 Flurometer (Life Technologies, CA, USA).;</p>
<br />

<a name="Library_Preparation"></a>
<h3>1.2 Library Preparation</h3>

<p class="paragraph">The genomic DNA is randomly fragmented to size of 350bp, and then DNA fragments were end polished, A-tailed, and ligated with the  adapters of Illumina sequencing, and further PCR enriched with primers of P5 and P7 oligos. The PCR products as final construct of the libraries were purified (AMPure XP system), followed by quality control test that included size distribution by Agilent 2100 Bioanalyzer (Agilent Technologies, CA, USA), and molarity measurement using real-time PCR. The experimental procedures of DNA library preparation are shown in <strong>Figure 1.2</strong>.</p>
<p class="center">
<img src="src/images/wgs_experiment_pipeline.png" />
</p>

<p class="name_fig">Figure 1.2 Experimental procedures of library preparation</p>

<a name="Clustering_and_Sequencing"></a>
<h3>1.3 Clustering and Sequencing</h3>
<p class="paragraph">If the library qualifies, it will be sequenced on an Illumina platform according to effective concentration and the expected data volume. </p>

</div>
<!---------------------------Experimental Process Over--------------------------------------------------->

<!---------------------------Bioinformatics analysis process--------------------------------------------->
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Bioinformatics_Analysis_Procedures">Novogene Co., Ltd</a>
<hr />
</p><br />
<h2>2 Bioinformatics Analysis Pipeline</h2>
<p class="paragraph">This is the workflow for plant and animal whole genome resequencing data of advanced bioinformatic analysis with a well-annotated reference genome, as follows:</p>






    
    
    
    
    
    <p class="center">
    <img  class="small6" src="src/images/flow_snpindel.png" />
    </p>
    

<p class="name_fig">Figure 2.1 Bioinformatics analysis pipeline</p>

</div>
<!---------------------------Bioinformatics analysis process over--------------------------------------------->

<!-- ------------------------------------Result Raw Data------------------------------------------>
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Results_of_Analyses">Novogene Co., Ltd</a>
<hr />
</p>

<h2>3 Project Results</h2>

<a name="Raw_Data"></a>
<h3>3.1 Raw Data</h3>
<p class="paragraph">Original image data file from high-throughput sequencing platforms (like Illumina) is transformed to sequenced reads (called Raw Data or Raw Reads) by CASAVA base recognition (Base Calling). Raw data are stored in FASTQ (.fq) format files, which contain sequencing reads and corresponding base quality. Each read has four descriptive lines, as indicated below:</p>

<p class="seq">
@K00124:82:H2MH5BBXX:1:1101:31389:1158 2:N:0:0<br/>
TAGCCACATAGAAACCAACAGCCATATAACTGGTAGCTTTAAGCGGCTCACCTTTAGCATCAACAGGCCACAACCAACCAGAACGTGAAAAAGCGTCCTGCGTGTAGCGAACTGCGATGGGCATACAGATCGGAAGAGCGTCGTGTAGGG<br/>
+<br/>
AAFFFKKKKKKKKFKKKFFKKAAFKKKKKFKKKKFKKA,FKKKKKKKKKAKKFKKKKKKKAKKKKKKFFKKKKF&lt;FFKKKKKKKKKKKKKFKKFKKF7FFFFFKFKKKFKKKKKKKKF&lt;FFKKKKFKKKKKFKFKFKKFK&lt;&lt;F,A7,AFK
</p>
<p class="tremark">Line 1: begins with the at sign (@), followed by sequence identifiers and an optional description (such as FASTA header).<br />Line 2: base sequences (raw read, A, G, C, and T)<br />Line 3: begins with the plus sign (+), optionally followed by the same Illumina sequence identifiers and description information as in Line 1.<br />Line 4: the quality values for each base, corresponding to the data in Line 2.<br />
</p>

<div id="tb">
<p class="center">
<p class="name_fig">Table 3.1 Illumina sequence identifier details</p>
<table class="gy">
<tr><td>K00124</td><td>Unique instrument name</td></tr>
<tr><td>82</td><td>Run ID</td></tr>
<tr><td>H2MH5BBXX</td><td>Flowcell ID</td></tr>
<tr><td>1</td><td>Flowcell lane</td></tr>
<tr><td>1101</td><td>Tile number within the flowcell lane</td></tr>
<tr><td>31389</td><td>'x'-coordinate of the cluster within the tile</td></tr>
<tr><td>1158</td><td>'y'-coordinate of the cluster within the tile</td></tr>
<tr><td>2</td><td>Member of a pair, 1 or 2 (paired-end or mate-pair reads only)</td></tr>
<tr><td>N</td><td>Y if the read fails filter (read is bad), N otherwise</td></tr>
<tr><td>0</td><td>0 when none of the control bits are on, otherwise it is an even number</td></tr>
<tr><td>ATCACG</td><td>Index sequence</td></tr>
</table>
</p>
</div>
<!-- ------------------------------------Result Raw Data over------------------------------------------>

<!-- ------------------------------------Result QC------------------------------------------>
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Sequencing_Data_Quality_Control">Novogene Co., Ltd</a>
<hr />
</p>
<h3>3.2 Data Quality Control</h3>

<a name="Sequencing_Quality_Distribution"></a>
<h4>3.2.1　Sequencing Quality Distribution</h4>
<p class="paragraph">If the sequencing error rate is represented by e, and Illumina sequencing quality by Q<sub>phred</sub>, the quality score of a base (Phred score) is calculated by the following equation: Q<sub>phred</sub>=-10log<sub>10</sub>(e). The correspondence relationship between Illunima sequencing quality and Phred score in base calling by Casava version 1.8 is listed as follows:</p>
<p class="name_fig">Table 3.2 Sequencing error rate and corresponding base quality value</p>
<table class="gy">
<tr><th>Phred score</th><th>Error Rate</th><th>Correct Rate</th><th>Q-score</th></tr>
<tr><td>10</td><td>1/10</td><td>90%</td><td>Q10</td></tr>
<tr><td>20</td><td>1/100</td><td>99%</td><td>Q20</td></tr>
<tr><td>30</td><td>1/1000</td><td>99.9%</td><td>Q30</td></tr>
<tr><td>40</td><td>1/10000</td><td>99.99%</td><td>Q40</td></tr>
</table>

<p class="paragraph">For next-generation sequencing (NGS), the sequencing platform, chemical reactants, and sample quality can influence sequencing quality and base error rate. Sequencing quality distribution is examined over the full length of all sequences, to detect any sites (base positions) with an unusually low sequencing quality, where incorrect bases may be incorporated at abnormally high levels. For detailed sequencing quality distribution, please refer to <strong>Figure 3.1</strong>.
</p>



<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure1_1_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure1_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure1_1_pane_ERBAS_H2LGJDSXY_L1" role="tabpanel" aria-labelledby="gallery_figure1_1_list_ERBAS_H2LGJDSXY_L1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/Quality/ERBAS_H2LGJDSXY_L1.QM.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_ERBAS_H2KNJDSXY_L1" role="tabpanel" aria-labelledby="gallery_figure1_1_list_ERBAS_H2KNJDSXY_L1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/Quality/ERBAS_H2KNJDSXY_L1.QM.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure1_1_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure1_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure1_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_ERBAS_H2LGJDSXY_L1" data-toggle="list" href="#gallery_figure1_1_pane_ERBAS_H2LGJDSXY_L1" role="tab" aria-controls="ERBAS_H2LGJDSXY_L1">ERBAS_H2LGJDSXY_L1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_ERBAS_H2KNJDSXY_L1" data-toggle="list" href="#gallery_figure1_1_pane_ERBAS_H2KNJDSXY_L1" role="tab" aria-controls="ERBAS_H2KNJDSXY_L1">ERBAS_H2KNJDSXY_L1</a>

</div>
</div>
</div>	
</p>






<p class="name_fig">Figure 3.1 Distribution of sequencing quality</p>
<p class="premark">The x-axis shows the base position within a sequencing read, and the y-axis shows the average phred score of all reads at each position.<br />(Pair-end sequencing data are plotted together, with the first PE150 bp representing read 1 and the following PE150 bp for read 2.)</p>
</div>

<!----------------------------------------- Distribution of Sequencing Errors -------------------------------------->
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Distribution_of_Sequencing_Errors">Novogene Co., Ltd</a>
<hr />
</p><br />
<h4>3.2.2 Sequencing Error Rate</h4>
<p class="paragraph">Sequencing error rate and base quality can be affected by various factors such as sequencing platform, chemical reagent and sample quality. Due to the consumption of chemical reagents, error rate is increasing with read extension, which is a common feature of Illumina high throughput sequencing platforms. The sequencing error rate distribution can be featured as below: </p>
<p class="tremark">(1) Error rate increases with the sequencing reads for consumption of sequencing reagent. It is common in the Illumina high-throughput sequencing platform.<br />(2) The sequencing error rate is higher for the first several bases than at other positions, which is likely the result of reading errors during the first few cycles after calibration of the optical instruments.<br /></p>

<br />

	<p class="center">
	<div class="row gallery-row">
	<div class="col-1">
	<div id="gallery_figure_error_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_error_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_error_pane_ERBAS_H2LGJDSXY_L1" role="tabpanel" aria-labelledby="gallery_figure_error_list_ERBAS_H2LGJDSXY_L1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/Error/ERBAS_H2LGJDSXY_L1.Error.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_error_pane_ERBAS_H2KNJDSXY_L1" role="tabpanel" aria-labelledby="gallery_figure_error_list_ERBAS_H2KNJDSXY_L1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/Error/ERBAS_H2KNJDSXY_L1.Error.png" /></div>
	
	</div>
	</div>
	<div class="col-1">
	<div id="gallery_figure_error_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_error_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_error_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_error_list_ERBAS_H2LGJDSXY_L1" data-toggle="list" href="#gallery_figure_error_pane_ERBAS_H2LGJDSXY_L1" role="tab" aria-controls="ERBAS_H2LGJDSXY_L1">ERBAS_H2LGJDSXY_L1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_error_list_ERBAS_H2KNJDSXY_L1" data-toggle="list" href="#gallery_figure_error_pane_ERBAS_H2KNJDSXY_L1" role="tab" aria-controls="ERBAS_H2KNJDSXY_L1">ERBAS_H2KNJDSXY_L1</a>
	
	</div>
	</div>
	</div>

	</p>




<p class="name_fig">Figure 3.2 Distribution of sequencing error rate</p>
<p class="premark">The x-axis shows the base position along each sequencing read and the y-axis shows the base error rate.<br />(Pair-end sequencing data are plotted together, with the first PE150 bp representing read 1 and the following PE150 bp for read 2.)</p>
</div>

<!-----------------------------------------Sequencing Data Filtration-------------------------------------->
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Sequencing Data Filtration">Novogene Co., Ltd</a>
<hr />
</p><br />
<h4>3.2.3 Sequencing Data Filtration</h4>
<p class="paragraph">The sequencing reads/raw reads often contain low quality reads or reads with adaptors, which will affect the quality of downstream analysis. To avoid this, it is necessary to filter the raw reads and obtain clean reads. Raw reads filtering is as follows:</p>
<p class="list">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Remove the paired reads when either read contains adapter contamination;<br />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Remove the paired reads when uncertain nucleotides (N) constitute more than 10 percent of either read;<br />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Remove the paired reads when low quality nucleotides (base quality less than 5, Q ≤ 5) constitute more than 50 percent of either read.</p>


	<p class="center">
	<div class="row gallery-row">
	<div class="col-1">
	<div id="gallery_figure1_2_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure1_2_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure1_2_pane_ERBAS_H2LGJDSXY_L1" role="tabpanel" aria-labelledby="gallery_figure1_2_list_ERBAS_H2LGJDSXY_L1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/Class/ERBAS_H2LGJDSXY_L1.pie3d.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure1_2_pane_ERBAS_H2KNJDSXY_L1" role="tabpanel" aria-labelledby="gallery_figure1_2_list_ERBAS_H2KNJDSXY_L1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/Class/ERBAS_H2KNJDSXY_L1.pie3d.png" /></div>
	
	</div>
	</div>
	<div class="col-1">
	<div id="gallery_figure1_2_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure1_2_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure1_2_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_ERBAS_H2LGJDSXY_L1" data-toggle="list" href="#gallery_figure1_2_pane_ERBAS_H2LGJDSXY_L1" role="tab" aria-controls="ERBAS_H2LGJDSXY_L1">ERBAS_H2LGJDSXY_L1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_ERBAS_H2KNJDSXY_L1" data-toggle="list" href="#gallery_figure1_2_pane_ERBAS_H2KNJDSXY_L1" role="tab" aria-controls="ERBAS_H2KNJDSXY_L1">ERBAS_H2KNJDSXY_L1</a>
	
	</div>
	</div>
	</div>

	</p>


<p class="name_fig">Figure 3.3 Classification of the sequenced reads</p>
<p class="premark">
	(1) Adapter related: (reads containing adapter) / (total raw reads).<br />
	(2) Containing N: (reads with more than 10% N) / (total raw reads).<br />
	(3) Low quality: (reads of low quality) / (total raw reads).<br />
	(4) Clean reads: (clean reads) / (total raw reads).</p>
</div>

<!-----------------------------------------GC Content-------------------------------------->


<!-----------------------------------------Statistics_of_Sequencing_Data-------------------------------------->
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Statistics_of_Sequencing_Data">Novogene Co., Ltd</a>
<hr />
</p><br />

<h4>3.2.4 Statistics of Sequencing Data</h4>
<p class="paragraph">According to the sequencing feature of Illumina platforms, for paired-end sequencing data we require that Q30 (the percent of bases with phred-scaled quality scores greater than 30) should be above 80%. Statistics of sequencing data are listed in <strong>Table 3.3</strong>.</p>
<p class="name_fig">Table 3.3 Statistics of Sequencing Data</p>



<p class="center">
<div id="tb">
<table class="gy">
<tr><th>Sample name</th><th>Lane</th><th>Raw reads</th><th>Raw data (G)</th><th>Clean data (G)</th><th>Effective (%)</th><th>Error (%)</th><th>Q20 (%)</th><th>Q30 (%)</th><th>GC (%)</th></tr>

    
        
            <tr>
            <td>ERBAS</td><td>H2LGJDSXY_L1</td><td>18650829</td><td rowspan=2>16.5</td><td rowspan=2>16.4</td><td>99.86</td><td>0.03</td><td>98.47</td><td>94.94</td><td>35.47</td>
            </tr>
        
    
        
            <tr>
            <td>ERBAS</td><td>H2KNJDSXY_L1</td><td>36229380</td><td>99.9</td><td>0.03</td><td>97.14</td><td>92.00</td><td>35.22</td>
            </tr>
        
    

</table>
</div>
</p>

<p class="paragraph">The details for the sequencing data statistics are as follows:</p>
<p class="tremark">(1) Sample name: Sample name.<br />
	(2) Lane: The flowcell ID and lane number during the sequencing (FlowcellID_LaneNumber).<br />
	(3) Raw reads: The number of sequencing reads pairs; four lines will be considered as one unit according to FASTQ format. <br />
	(4) Raw data (G): The original sequence data volume.<br />
	(5) Clean data (G): The sequence data volume calculated by clean data..<br />
	(6) Effective (%): The ratio of clean data to raw data.<br />
	(7) Error (%): Overall error rate of base.<br />
	(8) Q20 (%): The percentage of bases with higher Phred score than 20.<br />
	(9) Q30 (%): The percentage of bases with higher Phred score than 30.<br />
	(10) GC: The percentage of G and C in the total bases.<br /></p>
<br />


</div>
<!------------------------------------Statistics_of_Sequencing_Data over----------------------------------->

<!------------------------------------Sequencing Evaluation Q&A----------------------------------->
<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Sequencing_Evaluation_Q&A">Novogene Co., Ltd</a>
<hr />
</p><br />


<h4>3.2.5 Sequencing Evaluation Q&A </h4>

<div class="QA">
	<table width=100% align="center" cellspacing="0" cellpadding="8">
		<tr><td class="question"><b>Q.: </b>As the sequencing error increases with the read length, what is the acceptable range of sequencing error rate?</td></tr>
		<tr><td class="answer"><b>A.: </b>Currently for Illumina sequencers, the per base error rate is generally lower than 1%, and the highest acceptable threshold is 6%.</td></tr>
		<tr><td class="question"><b>Q.: </b>What is the data filtering criterion at Novogene? Is it strictly adhered?</td></tr>
		<tr><td class="answer"><b>A.: </b>Novogene utilizes stringent data quality control procedures and strictly adheres to the high standard to guarantee the accuracy and reliability of the sequencing data. The detailed data quality control criterion is as follows:<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(1)</b> Discard the adapter-containing reads;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(2)</b> Discard the paired reads when uncertain nucleotides (N) constitute more than 10 percent of either read;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(3) </b>Discard the paired reads when low quality nucleotides (base quality less than 5, Q ≤ 5) constitute more than 50 percent of either read.</td></tr>
		<tr><td class="question"><b>List of Related Terms</b>:</td></tr>
		<tr><td class="answer">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>adapter</b>: the oligo nucleotides ligated to sample DNA at library preparation, for proper adhesion to the flow-cell via base-pairing in DNA sequencing.</td></tr>
		<tr><td class="answer">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>index</b>: the unique sequence tag for distinguishing each individual sample from multiplexing.</td></tr>
		<tr><td class="answer">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>Q20,Q30</b>: the proportion of bases with Phred score higher than 20 or 30. The Phred score, which is negatively correlated to the probability of incorrect base-calling, is calculated by the equation (Q<sub>phred</sub>=-10log<sub>10</sub>(e)) of sequencing error rate (e), indicating the sequencing quality.</td></tr>
		<tr><td class="answer">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>raw data/raw reads</b>: the original sequence data output by the instrument for a specific sample.</td></tr>
		<tr><td class="answer">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>clean data/clean reads</b>: the output of data from filtering raw data with quality control, which will be used for further analysis.</td></tr>
		</table>
</div>
</div>
<!------------------------------------Sequencing Evaluation Q&A over----------------------------------->
<!----------------------------------------Mapping Statistics----------------------------------------->

<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Mapping_Statistics">Novogene Co., Ltd</a>
<hr />
</p><br />
<h3>3.3 Sequence Alignment</h3>
<p class="paragraph">The effective sequencing data was aligned with the reference sequence through BWA<SUP>[2]</SUP> software (parameters: mem -t 4 -k 32 -M), and the mapping rate and coverage were counted according to the alignment results (see Table 3.4).</p>
<br />

<a name="Statistics_of_Reference_Genome"></a>
<h4>3.3.1　Statistics of Reference Genome</h4>

<p class="paragraph">Reference genome is downloaded from: <a href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/625/215/GCF_001625215.1_ASM162521v1/GCF_001625215.1_ASM162521v1_genomic.fna.gz">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/625/215/GCF_001625215.1_ASM162521v1/GCF_001625215.1_ASM162521v1_genomic.fna.gz</a></p>

<p class="name_fig">Table 3.4 Statistics of Reference Genome</p>
<p class="center">
<div id="tb">
<table class="gy" align="center">
<tr><th>Seq number</th><th>Total length</th><th>GC content (%)</th><th>Gap rate (%)</th><th>N50 length</th><th>N90 length</th></tr>

	<tr>
	<td>4,826</td><td>421,539,040</td><td>34.79</td><td>8.24</td><td>36,610,139</td><td>514,035</td>
	</tr>

</table>
</div>
</p>
<p class="paragraph">The detailed information of reference are as follows: </p>
<p class="tremark">
	(1) Seq number: the total number of the assembled genomic sequences.<br />
	(2) Total length: the total length of the assembled genomic sequence.<br />
	(3) GC content: the GC content of the reference genome.<br />
	(4) Gap rate: the proportion of unknown sequence (N) in the reference genome assembly.<br />
	(5) N50 length: the length of scaffold N50, of which 50% of the sequence is higher than this level.<br />
	(6) N90 length: the length of scaffold N90, of which 90% of sequence is higher than this level.<br />
	</p>
<br />
</div>

<!------------------------Mapping Statistics with Reference Genome--------------------------  -->
<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Mapping_Statistics_with_Reference_Genome">Novogene Co., Ltd</a>
<hr />
</p><br />
<h4>3.3.2　Mapping Statistics with Reference Genome</h4>
<p class="paragraph">The mapping rates of samples reflect the similarity between each sample and the reference genome. The depth and coverage are indicators of the evenness and homology with the reference genome.</p>

<p class="name_fig">Table 3.5 Statistics of mapping rate, depth and coverage</p>
<p class="center">
<div id="tb">
<table class="gy" align="center">
<tr><th>Sample</th><th>Mapped reads</th><th>Total reads</th><th>Mapping rate (%)</th><th>Average depth(X)</th><th>Coverage at least 1X (%)</th><th>Coverage at least 4X (%)</th></tr>

	<tr>
	<td>ERBAS</td><td>17,658,530</td><td>109,637,186</td><td>16.11</td><td>34.65</td><td>10.92</td><td>6.06</td>
	</tr>

</table>
</div>
</p>

<p class="paragraph">The details for mapping statistics are as follows: </p>
<p class="tremark">
(1) Sample: Sample names.<br />
(2) Mapped reads: The number of clean reads mapped to the reference assembly, including both single-end reads and reads in pairs.<br />
(3) Total reads: Total number of effective reads in clean data.<br />
(4) Mapping rate: The ratio of the reference genome assembly mapped reads to the total sequenced clean reads.<br />
(5) Average depth: The average depth of mapped reads at each site, calculated by the total number of bases in the mapped reads dividing by size of the assembled genome.<br />
(6) Coverage at least 1X: The percentage of the assembled genome with more than one read at each site.<br />
(7) Coverage at least 4X: The percentage of the assembled genome with ≥4X coverage at each site.<br />
</p>

<a name="Mapping_Summary"></a>
<h4> 3.3.3 Mapping Summary </h4>

<p class="paragraph">For the current 421,539,040 bp reference genome, the mapping rate of this sample is 16.11%. Refer to the reference genome (without Ns),the average depth is 34.65X, and more than 1X coverage reaches 10.92%. This result is in the qualified normal range and may serve in the subsequent variation detection and related analyses.</p>



<a name="Mapping_Q&A"></a>
<h4> 3.3.4 Mapping Q&A </h4>
<div class="QA">
<table width=100% align="center" cellspacing="0" cellpadding="8">
<tr><td class="question"><b>Q.: </b>Which files are required for reference genome mapping?</td></tr>
<tr><td class="answer"><b>A.: </b>As for whole genome sequencing (WGS), only the corresponding reference genome file in FASTA format is required, while the whole exome sequencing (WES) also needs the target region file in BED format.</td></tr>
<tr><td class="question"><b>Q.: </b>What are the potential causes of low mapping rate?</td></tr>
<tr><td class="answer"><b>A.: </b>The three major causes are: (1) with poorly assembled reference genome or a relatively far genetic relationship between the reference and the sample; (2) treatments to DNA that may changed the sequence (e.g. bisulfite treatment) and herein affecting the mapping rate and (3) contamination of other species.</td></tr>
<tr><td class="question"><b>Q.: </b>Could the full-length reads be mapped to the reference? or should we trim from the sequence ends?</td></tr>
<tr><td class="answer"><b>A.: </b>According to the standard Illumina pair-end (PE) sequencing protocol, DNA fragments were ligated with adapters at both ends. The adapter habors universal sequences for flow-cell binding, DNA sequencing, as well as unique index located upstream of the sequencing primer for multiplexing. As for the PEPE150 sequencing of 350 bp library, the sequenced reads of PE150 bp at either end are adapter-free, which could be directly subjected to quality control for low quality reads filtration. The retaining sequences in PE150 bp length (namely clean data) are qualified for mapping with the reference genome.</td></tr>
</table>
</div>
</div>



<!-----------------------SNP Detection & Annotation------------------------------------>
<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="SNP_Detection_&_Annotation">Novogene Co., Ltd</a>
<hr />
</p><br />
<h3>3.4 SNP Detection & Annotation</h3>

<p class="paragraph">Single nucleotide polymorphism (SNP) refers to a variation in a single nucleotide which may occur at some specific position in the genome, including transition and transversion of a single nucleotide. We detected the individual SNP variations using SAMtools<sup>[3]</sup> with the following parameter: 'mpileup -m 2 -F 0.002 -d 1000'.</p>
<p class="paragraph">To reduce the error rate in SNP detection, we filtered the results with the criterion as follows:</p>
<p class="list">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) The number of support reads for each SNP should be more than 4;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) The mapping quality (MQ) of each SNP should be higher than 20;</p>



<a name="Statistics_of_SNP_Detection_&_Annotation"></a>
<h4>3.4.1 Statistics of SNP Detection & Annotation</h4>
<p class="paragraph">ANNOVAR<SUP>[4]</SUP> is a widely used software in variation annotation with multiple capabilities, including gene-based annotation, region-based annotation, filter-based annotation as well as other functionalities. Novogene use ANNOVAR to do annotation of detected SNPs.</p>
<p class="name_fig">Table 3.6 Statistics of SNP detection and annotation</p>
<p class="center">
<div id="tb">
<table class="gy" align="center">
<tr><th rowspan='2'>Sample</th><th rowspan='2'>Upstream</th><th colspan='4'>Exonic</th><th rowspan='2'>Intronic</th><th rowspan='2'>Splicing</th><th rowspan='2'>Downstream</th><th rowspan='2'>Upstream/Downstream</th><th rowspan='2'>Intergenic</th><th rowspan='2'>ts</th><th rowspan='2'>tv</th><th rowspan='2'>ts/tv</th><th rowspan='2'>Het rate(‰)</th><th rowspan='2'>Total</th></tr>
<tr><th>Stop gain</th><th>Stop loss</th><th>Synonymous</th><th>Non-synonymous</th></tr>

	<tr>
	<td>ERBAS</td><td>7395</td><td>1696</td><td>57</td><td>416768</td><td>197589</td><td>64538</td><td>215</td><td>7951</td><td>1054</td><td>189179</td><td>580746</td><td>344394</td><td>1.686</td><td>0.542</td><td>925140</td>
	</tr>

</table>
</div>
</p>
<p class="paragraph">The details for SNP detection and annotation statistics are as follows:</p>
<p class="tremark">
(1) Sample: Sample name;<br />
(2) Upstream: SNPs located within 1 kb upstream (away from transcription start site) of the gene.<br />
(3) Exonic: SNPs located in exonic region; Non-synonymous: single nucleotide mutation with changing amino acid sequence; Stop gain/loss: a nonsynonymous SNP that leads to the introduction/removal of stop codon at the variant site; Synonymous: single nucleotide mutation without changing amino acid sequence;<br />
(4) Intronic: SNPs located in intronic region;<br />
(5) Splicing: SNPs located in the splicing site (2 bp range of the intron/exon boundary).<br />
(6) Downstream: SNPs located within 1 kb downstream (away from transcription termination site) of the gene region.<br />
(7) Upstream/Downstream: SNPs located within the &lt 2 kb intergenic region, which is in 1 kb downstream or upstream of the genes.<br />
(8) Intergenic: SNPs located within the &gt 2 kb intergenic region.<br />
(9) ts: Transitions, a point mutation that changes a purine nucleotide to another purine (A ↔ G) or a pyrimidine nucleotide to another pyrimidine (C ↔ T). Approximately two out of three SNPs are transitions.<br />
(10) tv: Transversions, the substitution of a (two ring) purine for a (one ring) pyrimidine or <i>vice versa</i>.<br />
(11) ts/tv: The ratio of transitions to transversions.<br />
(12) Het rate: Genome-wide heterozygous rate, calculated by the ratio of heterozygous SNPs to the total number of genome bases.<br />
(13) Total: The total number of SNPs.<br />
</p>
</div>


<!-- --------------------------------------SNP Quality Distribution--------------------------------------- -->
<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="SNP_Quality_Distribution">Novogene Co., Ltd</a>
<hr />
</p><br />
<h4>3.4.2 SNP Quality Distribution</h4>
<p class="paragraph">To assess the credibility of detected SNPs, we checked the distribution of support reads number, SNP quality, as well as the distance between adjacent SNPs. The results are shown in <strong>Figure 3.4</strong>.</p>


	<p class="center">
	<div class="row gallery-row">
	<div class="col-1">
	<div id="gallery_figure_snp_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_snp_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_snp_pane_SNP_readsNum_cumulative_distribution" role="tabpanel" aria-labelledby="gallery_figure_snp_list_SNP_readsNum_cumulative_distribution" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/SNP/SNP_readsNum_cumulative_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_snp_pane_SNP_distance_cumulative_distribution" role="tabpanel" aria-labelledby="gallery_figure_snp_list_SNP_distance_cumulative_distribution" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/SNP/SNP_distance_cumulative_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_snp_pane_SNP_quality_cumulative_distribution" role="tabpanel" aria-labelledby="gallery_figure_snp_list_SNP_quality_cumulative_distribution" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/SNP/SNP_quality_cumulative_distribution.png" /></div>
	
	</div>
	</div>
	<div class="col-1">
	<div id="gallery_figure_snp_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_snp_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_snp_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_snp_list_SNP_readsNum_cumulative_distribution" data-toggle="list" href="#gallery_figure_snp_pane_SNP_readsNum_cumulative_distribution" role="tab" aria-controls="SNP_readsNum_cumulative_distribution">SNP_readsNum_cumulative_distribution</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_snp_list_SNP_distance_cumulative_distribution" data-toggle="list" href="#gallery_figure_snp_pane_SNP_distance_cumulative_distribution" role="tab" aria-controls="SNP_distance_cumulative_distribution">SNP_distance_cumulative_distribution</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_snp_list_SNP_quality_cumulative_distribution" data-toggle="list" href="#gallery_figure_snp_pane_SNP_quality_cumulative_distribution" role="tab" aria-controls="SNP_quality_cumulative_distribution">SNP_quality_cumulative_distribution</a>
	
	</div>
	</div>
	</div>

	</p>


<p class="name_fig">Figure 3.4 Cumulative distribution of SNP quality</p>
<p class="premark">Note: These figures show the quality distribution of SNPs by, from top to bottom, the distribution of SNP support reads number, the distribution of distances between adjacent SNPs, and the cumulative distribution of SNP quality.</p>
</div>



<!-- --------------------------------------SNP Mutation Frequency--------------------------------------- -->
<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="SNP_Mutation_Frequency">Novogene Co., Ltd</a>
<hr />
</p><br />
<h4>3.4.3 SNP Mutation Frequency</h4>
<p class="paragraph">Take the T:A>C:G mutations as an example, this category includes mutations from T to C and A to G. When T>C mutation appears on either of the double-strand, the A>G mutation will be found in the same position of the other chain. Therefore the T>C and A>G mutations are classified into one category. Accordingly, the whole-genome SNP mutations could be classified into six categories. The frequency of each type is shown in <strong>Figure 3.5</strong>.</p>
<p class="center">
<a id="example2" href="src//pictures/SNP/SNP_frequency.png"><img class="normal1" src="src//pictures/SNP/SNP_frequency.png" alt="example2"/></a>
</p>
<p class="name_fig">Figure 3.5 Frequency of SNP mutations</p>
<p class="premark">The x-axis represents the number of the SNPs, and y-axis indicates the mutation types.</p>

<a name="SNP_Detection_&_Annotation_Q&A"></a>
<h4>3.4.4 SNP Detection & Annotation Q&A </h4>
<div class="QA">
<table width=100% align="center" cellspacing="0" cellpadding="8">
<tr><td class="question"><b>Q.: </b>What is the MQ quality for a SNP?</td></tr>
<tr><td class="answer"><b>A.: </b>The SNP quality is represented by the mapping quality of covering reads, calculated by the root-mean-square of the support reads' mapping quality.</td></tr>
<tr><td class="question"><b>Q.: </b>What is QUAL value for a SNP?</td></tr>
<tr><td class="answer"><b>A.: </b>The QUAL value is the Phred quality score (QUAL), represents the probability (<i>p</i>) of SNP truly existing at certain position. The higher the QUAL value, the more likely the SNP exists. The relationship between QUAL and <i>p</i> is QUAL=-10*log<sub>10</sub> (1-<i>p</i>). Therefore, the QUAL value of 20 means the probability of the existence of this SNP is 99%.</td></tr>
<tr><td class="question"><b>Q.: </b>What are transitions and transversions?</td></tr>
<tr><td class="answer"><b>A.: </b>Transitions refers to the changes between A and G, which are both purines, or between C and T, which are both pyrimidines; while transversions reprents changes between a purine and a pyrimidine, such as between A and T.</td></tr>
<tr><td class="question"><b>Q.: </b>What is the heterozygous SNP?</td></tr>
<tr><td class="answer"><b>A.: </b>Heterozygous SNPs are those called with both REF (the same to reference) and ALT (different from reference) genotypes in a diploid species.</td></tr>

<tr><td class="question"><b>Q.: </b>How to verify the SNP genotypes?</td></tr>
<tr><td class="answer"><b>A.: </b>The "golden standard" of SNP verification is PCR amplification followed by Sanger sequencing.</td></tr>
<tr><td class="question"><b>Q.: </b>If the PCR-sequencing method failed to verify the detected SNP, did this mean that the NGS SNP calling is not reliable?</td></tr>
<tr><td class="answer"><b>A.: </b>
SNP calling in NGS is based on the support reads and depended on the sufficient coverage depth, which ensures the accuracy of most but not all the detected SNPs. We recommend to double-check the PCR results first, then use a genome browser such as <i>Savant</i> and <i>IGV</i> to manually check the mapped reads of the NGS result.</td></tr>
</table>
</div>

</div>


<!-------------------------------------InDel Detection & Annotation------------------------------------->
<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="InDel_Detection_&_Annotation">Novogene Co., Ltd</a>
<hr />
</p><br />
<h3>3.5 InDel Detection & Annotation</h3>

<p class="paragraph">InDel refers to the insertion or deletion of ≤ 50 bp sequences in the DNA. Novogene uses SAMTOOLS<sup>[3]</sup> with the following parameter 'mpileup -m 2 -F 0.002 -d 1000' to detect InDels and followed by annotation using ANNOVAR<sup>[4]</sup>. The filter conditions are same with SNPs.</p>



<a name="Statistics_of_InDel_Detection_&_Annotation"></a>
<h4>3.5.1 Statistics of InDel Detection & Annotation</h4>
<p class="name_fig">Table 3.7 Statistics of InDel detection and annotation</p>
<p class="center">
<div id="tb">
<table class="gy" align="center">
<tr><th rowspan='2'>Sample</th><th rowspan='2'>Upstream</th><th colspan='6'>Exonic</th><th rowspan='2'>Intronic</th><th rowspan='2'>Splicing</th><th rowspan='2'>Downstream</th><th rowspan='2'>Upstream/Downstream</th><th rowspan='2'>Intergenic</th><th rowspan='2'>Insertion</th><th rowspan='2'>Deletion</th><th rowspan='2'>Het rate(‰)</th><th rowspan='2'>Total</th></tr>
<tr><th>Stop gain</th><th>Stop loss</th><th>Frameshift deletion</th><th>Frameshift insertion</th><th>Non-frameshift deletion</th><th>Non-frameshift insertion</th></tr>

	<tr>
	<td>ERBAS</td><td>709</td><td>131</td><td>4</td><td>77</td><td>40</td><td>212</td><td>257</td><td>3572</td><td>41</td><td>862</td><td>65</td><td>30898</td><td>1955</td><td>1479</td><td>0.006</td><td>31303</td>
	</tr>

</table>
</div>
</p>

<p class="paragraph">The details of InDel annotation statistics are as follows:</p>
<p class="tremark">
(1) Sample: Sample names.<br />
(2) Upstream: InDels located within 1 kb upstream (away from transcription start site) of the gene.<br />
(3) Exonic: InDels located in exonic region; Stop gain/loss: InDel that leads to the introduction/removal of stop codon at the variant site; Frameshift deletion/insertion: InDel mutation changing the open reading frame with deletion or insertion; Non-Frameshift deletion/insertion: 
InDel mutation without changing the open reading frame with deletion or insertion sequences of 3 or multiple of 3 bases;
<br />
(4) Intronic: InDel located in intronic region;<br />
(5) Splicing: InDel located in the splicing site (2 bp range of the intron/exon boundary).<br />
(6) Downstream: InDel located within 1 kb downstream (away from transcription termination site) of the gene region.<br />
(7) Upstream/Downstream: InDel SNPs located within the &lt 2 kb intergenic region, which is in 1 kb downstream or upstream of the genes.<br />
(8) Intergenic: InDel located within the > 2 kb intergenic region.<br />
(9) Het rate: InDel heterozygous rate, calculated by the ratio of InDels to the total number of genome bases.<br />
(10) Total: The total number of InDels.<br />
</p>
</div>

<!-------------------------Length Distribution of CDS-located InDels---------------------------------->
<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Length_Distribution_of_CDS-located_InDels">Novogene Co., Ltd</a>
<hr />
</p><br />
<h4>3.5.2 Length Distribution of CDS-located InDels</h4>
<p class="center">
<a id="example2" href="src/pictures/InDel/InDel_CDSpercentage.png"><img class="normal1" src="src/pictures/InDel/InDel_CDSpercentage.png" alt="example2"/></a>
</p>
<p class="name_fig">Figure 3.6 Length distribution of CDS-located InDels</p>
<p class="premark">The x-axis represents the proportion of the InDels with a certain length, and y-axis indicates the length of the InDels.</p>

<a name="InDel_Detection_&_Annotation_Q&A"></a>
<h4>3.5.3 InDel Detection & Annotation Q&A </h4>
<div class="QA">
<table width=100% align="center" cellspacing="0" cellpadding="8">
<tr><td class="question"><b>Q.: </b>What is the heterozygous rate for an InDel?</td></tr>
<tr><td class="answer"><b>A.: </b>Ratio of heterozygous InDels to total number of InDels, the heterozygous Indel is an Indel only located in one of the homologous chromosomes of a diploid sample.</td></tr>
<tr><td class="question"><b>Q.: </b>What's the difference between an InDel and a SNP?</td></tr>
<tr><td class="answer"><b>A.: </b>SNP mutation refers to the change from one nucleotide to another nucleotide (eg. A↔T), while InDel is a type of mutation with insertion or deletion of one or more nucleotides (eg, A↔AT, with a T insertion).</td></tr>
</table>
</div>
</div>




<!----------------------------------------SV Detection & Annotation-------------------------------->


<!-----------------------------------CNV Detection & Annotation-------------------------->



<!----------------------------------Visualization of Variation-------------------------------->


<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Visualization_of_Variation">Novogene Co., Ltd</a>
<hr />
</p><br />




<h3>3.6 Visualization of Variation</h3>
<p class="paragraph">For proper visualization of the variations on the whole-genome, the density distribution of SNP/InDel is drawn with Circos<SUP>[5]</SUP>.</p>



	<p class="center">
	<div class="row gallery-row">
	<div class="col-1">
	<div id="gallery_figure_circos_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_circos_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_circos_pane_ERBAS" role="tabpanel" aria-labelledby="gallery_figure_circos_list_ERBAS" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/Circos/ERBAS.png" /></div>
	
	</div>
	</div>
	<div class="col-1">
	<div id="gallery_figure_circos_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
	</div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_circos_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_circos_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_circos_list_ERBAS" data-toggle="list" href="#gallery_figure_circos_pane_ERBAS" role="tab" aria-controls="ERBAS">ERBAS</a>
	
	</div>
	</div>
	</div>
	</div>
	</p>



    
    <p class="name_fig">Figure 3.7 Whole genome variations distribution</p>
    <p class="premark">From outer to inner, chromosome, SNP, InDel.</p>
    

</div>





<!----------------------------------------Appendix----------------------------------------->

<div id="page">
<p class="head"><a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="Appendix">Novogene Co., Ltd</a>
<hr />
</p><br />

<h2>4 Appendix</h2>
<br />
<table width=95% align="center" id="result_note" cellspacing="0" cellpadding="5" class='tf2'>



		<tr><th colspan="3">1.Explanation on result file:</th></tr>

		<tr><td rowspan="2">compressed files in the fomat of *.tar.gz:</td><td>Unix/Linux/Mac user</td><td>use tar -zxvf *.tar.gz command</td></tr>

		<tr><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>

		<tr><td rowspan="2">compressed files in the fomat of *.gz:</td><td>Unix/Linux/Mac user</td><td>use gzip –d  *.gz command</td></tr>

		<tr><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>

		<tr><td rowspan="2">compressed files in the fomat of *.zip:</td><td>Unix/Linux/Mac user</td><td>use unzip *.zip command</td></tr>

		<tr><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>

</table>
<br />
<br />




<table width=95% align="center" id="result_note" cellspacing="0" cellpadding="5" class='tf2'>

		<tr><th colspan="3">2. Explanation on how to operating file:</th>

			</tr>

		<tr><td rowspan="2">*.fasta</td><td rowspan="2">sequence file, in the fomat of fasta. in genaral, because sequence of gene or genome is large, we provide customer with sampled*.fasta file(partial sequences of *.fasta). It will be more convenient to check file fomat.</td><td>unix/Linux/Mac user use less or more command can view sequences in the fomat of *.fasta.</td>

			</tr>

		<tr><td>windows user use editor Editplus/Notepad++ et al</td>

			</tr>

		<tr><td rowspan="2">*.fq/fastq</td><td rowspan="2">reads sequence file, in the fomat of fasta. it is no easy to open because of big file.</td><td>unix/Linux/Mac user use less or more command;</td>

			</tr>

		<tr><td>windows user use editor Editplus/Notepad++ et al</td>

			</tr>

		<tr><td rowspan="2">*.xls,*.txt</td><td rowspan="2">table result file; files are separated by(Tab)</td><td>unix/Linux/Mac user use less or more command</td>

			</tr>

		<tr><td>windows user use editor Editplus/Notepad++ et al, also can use Microsoft Excel to open.</td>

			</tr>

		<tr><td rowspan="2">*.png</td><td rowspan="2">figure result files; bitmap</td><td>unix/Linux/Mac user use display command to open.</td>

			</tr>

		<tr><td>windows user use picture viewer to view, for example photoshop et al</td>

		</tr>

		<tr><td>*.pdf</td><td>figure result files; vectorgraph, can be zoom in or out. it is very convenient for customer to view or edit. using Adobe Illustrator to edit,the picture can be used in paper for publishing.</td><td>windows/Mac user use Adobe Reader/Foxit reader/web browser to open, unix/Linux user use evince command to open.</td>

			</tr>

</table>
<br />




<a name="File catalog"></a>
<h3>4.1　Result Directory</h3>

<p class="paragraph">Click to open the result directory(html), and the corresponding file can be viewed directly when the report and the result files are stored in the same folder.</p>

<p class="paragraph">File catalog: <a href="src/result_tree.html" title="点击打开" target="_blank">html</a></p>




<a name="List_of_Softwares"></a>
<h3>4.2 List of Softwares</h3>
<p class="paragraph">Novogene provides the software list in bioinformatics analysis pipeline for your reference.</p>

<p class="center">
<p class="name_fig">Table 5.1 List of softwares in WGS analyses</p>
<div id="tb">
<table class="tf2">
<tr><th>Analysis</th><th>Software</th><th>Usage</th><th>Version</th></tr>

<tr><td style="text-align:left">&nbsp;&nbsp;Quality control </td><td>in-house</td><td>Quality control</td><td>1.0</td></tr>


<tr><td rowspan=3 style="text-align:left">&nbsp;&nbsp;Mapping</td><td>BWA</td><td>Mapping clean reads to the reference genome and generation of bam result files.</td><td>0.7.8-r455</td></tr>
<tr><td>SAMtools</td><td>Sorting the bam files and removing duplication reads.</td><td>0.1.19-44428cd</td></tr>
<tr><td>Picard</td><td>Merging the bam files from the same sample.</td><td>1.111</td></tr>



<tr><td style="text-align:left">&nbsp;&nbsp;SNP/InDel Detection</td><td>SAMtools</td><td>Detection and filtration of SNPs and InDels.</td><td>0.1.19-44428cd</td></tr>





<tr><td style="text-align:left">&nbsp;&nbsp;Variation Annotation</td><td>ANNOVAR</td><td>Annotation of the detected variations.</td><td>2015Mar22</td></tr>

</table>
</div>
</p>

</div>




<!----------------------------------------References----------------------------------------->
<div id="page">
<p class="head">
<a href="#home" title = "Home"><img class="logo" align="left" src="src/images/logo.png" /></a>
<a name="References">Novogene Co., Ltd</a>
<hr />
</p><br />
<h2>5 References</h2>
<p class="ref">

<p class="paragraph">[1] Cock, P.J.A., Fields, et al. The Sanger FASTQ file format for sequences with quality scores, and the Solexa/Illumina FASTQ variants. Nucleic Acids Research. 2010, 38(6):1767-1771.</p>


<p class="paragraph">[2] Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009, 25(14):1754-1760.</p>
<p class="paragraph">[3] Li H, Handsaker B, Wysoker A, et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics. 2009, 25(16):2078-2079.</p>



	
		
<p class="paragraph">[4] Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Research. 2010, 38(16):e164.</p>
			
<p class="paragraph">[5] Krzywinski M, Schein J, Birol I, et al. Circos: an information aesthetic for comparative genomics. Genome Research. 2009, 19(9):1639-1645.</p>
			
			
				
				
			
		
	


</p>
</div>


</body>
</html>

